ui/package.json

```diff
@@ -56,10 +56,10 @@
     "@mui/styles": "^5.11.2",
     "@paciolan/remote-component": "^2.11.0",
     "@redux-devtools/extension": "^3.2.3",
-    "@rjsf/core": "^5.0.0-beta.11",
+    "@rjsf/core": "5.0.0-beta.11",
     "@rjsf/material-ui": "^5.0.0-beta.11",
     "@rjsf/utils": "^5.0.0-beta.11",
-    "@rjsf/validator-ajv6": "^5.0.0-beta.11",
+    "@rjsf/validator-ajv6": "5.0.0-beta.11",
```

Comment:
I removed `^` because in CI environment due to caching issue it was installing latest version of `rjsf/core` and `rjsf/validator-ajv6` but we have special patch for these packages at `v5.0.0-beta.11` thus throwing build error.
This isn't seen in clean local env (with no `node_modules` and `package-lock.json`).
Though I have verified the behavior and everything works as it is supposed to be.


@Abhishek-kumar09 would you please have look once and verify in your env as well?

Reply:
@harkiratsm does this sound good?

Reply:
@harkiratsm no issues, right?

Reply:
No issues as we are on latest rjsf v5.1.0.